slacker.py

```diff
@@ -165,7 +170,7 @@ def channel_exists(self, channel_name):
     def delete_message(self, cid, message_timestamp):
         url_template = self.url + "chat.delete?token={}&channel={}&ts={}"
         url = url_template.format(self.token, cid, message_timestamp)
-        ret = self.session.get(url).json()
+        ret = self.session.post(url).json()
```

Comment:
@royrapoport Any reason these were GETs previously? If not, this is is working for me.

Reply:
Well, on the bright side, this one is dead code, but the other one, channel.archive is used quite a bit.